src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1614,11 +1616,18 @@ def test_webapp_hybridconnectionE2E(self, resource_group):
             JMESPathCheck('length(@)', 1),
             JMESPathCheck('[0].name', hyco_name)
         ])
+
+        self.cmd('webapp deployment slot create -g {} -n {} --slot {}'.format(
+            resource_group, webapp_name, slot_webapp_name))
+        self.cmd('webapp hybrid-connection add -g {} -n {} --namespace {} --hybrid-connection {} --slot {}'.format(
+            resource_group, webapp_name, namespace_name, hyco_name, slot_webapp_name))
```

Comment:
Is it possible to check these two commands output?

Reply:
there is a list call that checks this already.